getstarted.md

```diff
@@ -22,6 +22,70 @@ with Hadoop, as well as extensions to that foundation that make for totally
 awesome demos. You'll also find a few videos related to our work and of course,
 how to contact us!
 
+First, though, let's just take a look at how straightforward it is to get started with very complex software. You can do it during a lunch break:
+
+## A Lunch Break Project -- Visualize Earthquake Data with Zeppelin
+
+*This tutorial assumes that you have Juju 2.0 installed and bootstrapped. If you don't, try installing [the devel version of charmbox](https://github.com/juju-solutions/charmbox/tree/devel), setup a free cloud account via our [developer program](https://developer.juju.solutions), and follow the [Juju getting started guide](https://jujucharms.com/docs/devel/getting-started).*
+
+1. Deploy the hadoop processing bundle.
+```
+juju deploy hadoop-processing
+```
+1. Add the spark and apache-zeppelin charms. We'll deploy the latter as "zeppelin", to save us some typing later on.
+```
+juju deploy apache-zeppelin zeppelin
+juju deploy spark
+```
+1. Add some relations to connect everything together ("plugin" is our hadoop plugin charm, which makes it straightforward to connect applications to the core hadoop application).
+```
+juju relate openjdk spark
+juju relate spark plugin
+juju relate zeppelin spark
+```
+1. Expose the zeppelin port, so that you can talk to the web interface when it finishes setting itself up.
+```
+juju expose zeppelin
+```
+1. Wait for things to finish getting setup. This might be a good time to make yourself a sandwich, or spend some time making that perfect cup of espresso.
+```
+watch --color 'juju status --color'
+```
+1. Once your cluster is all green, take note of zeppelin's ip. You can get the ip by running juju status.
+```
+juju status zeppelin
+```
+1. Fetch [this rather cool Earthquake visualization notebook by Leemoon Soo from the Hortonworks gallery](https://raw.githubusercontent.com/hortonworks-gallery/zeppelin-notebooks/master/2APFTN3NY/note.json), and submit it to the zeppelin api.
+```
+wget https://raw.githubusercontent.com/hortonworks-gallery/zeppelin-notebooks/master/2APFTN3NY/note.json
+curl -H "content-type: application/json" -X POST --data @note.json http://<zeppelin ip>:9090/api/notebook
+```
+1. You can now open up Zeppelin's web interface in a browser and execute the notebook (click the "play" button near the top of the page).
+```
+sensible-browser https://<zeppelin ip>:9090
```

Comment:
bigtop zeppelin's port is 9080 instead of 9090.  just fyi in case we switch to that charm.